src/components/Button.js

```diff
@@ -0,0 +1,18 @@
+// import PropTypes from 'prop-types';
+import React from 'react';
+
+const Button = (props) => {
+  // eslint-disable-next-line react/prop-types
+  const { btnLabel } = props;
+  return (
+    <button className="calc-button" type="button">
+      {btnLabel}
+    </button>
+  );
+};
+
+// Button.propTypes = {
+//   btnType: PropTypes.string.isRequired,
+// };
+
+export default Button;
```

Comment:
go on and define the `btnLabel` propTypes as string, why comment ? it's in the requirement 
 - https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png